### PR TITLE
Add devfsadm for solaris.

### PR DESCRIPTION
**Pull request by notpeter**

### CODE REVIEW

File: pages/sunos/devfsadm.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# devfsadm
+
+> Administration command for /dev. Maintains the /dev namespace.
```
Comment (waldyrious) [inline pages/sunos/devfsadm.md]: I'd say format "/dev" using backticks, to highlight that it is a path. @igorshubovych any thoughts?

File: pages/sunos/devfsadm.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Scan for new disks:
+
+`devfsadm -c disk`
+
+- Cleanup any dangling /dev links and scan for new device:
+
+`devfsadm -C -v`
+
+- Dry-run, output what would be changed but make no modifications:
```
Comment (waldyrious) [inline pages/sunos/devfsadm.md]: instead of a comma, perhaps a dash would be more appropriate here.

**Comment by notpeter** [comment]
Agreed. I think backticking /dev is an odd corner case but helpful to highlight it's not just an arbitrary folder.

**Comment by igorshubovych** [comment]
LGTM

**Comment by waldyrious** [comment]
Thanks :)

**Pull request merged**